.gitignore

```diff
@@ -1,4 +1,5 @@
+.DS_Store
 node_modules
-*.log
-.idea
 dist
+.rpt2_cache
```

Comment:
what is it?

Reply:
https://www.npmjs.com/package/rollup-plugin-typescript2